ui/test/unit/registration/controllers/patientCommonController.spec.js

```diff
@@ -49,6 +51,30 @@ describe('PatientCommonController', function () {
 
     });
 
+    it('checks if the confirmation popup is prompted when the home button is clicked when config is true', function () {
+        scope.showSaveConfirmDialogConfig = true;
+        scope.onHomeNavigate = jasmine.createSpy("onHomeNavigate");
+        scope.confirmationPrompt = jasmine.createSpy("confirmationPrompt");
+        var element = angular.element("<a ng-click='onHomeNavigate()'>");
+        var compiled = $compile(element)(scope);
+        compiled.triggerHandler('click');
+        expect(scope.onHomeNavigate).toHaveBeenCalled();
+        expect(scope.showSaveConfirmDialogConfig).toBe(true);
+        scope.confirmationPrompt();
+        expect(scope.confirmationPrompt).toHaveBeenCalled();
+    });
+
+    it('checks if the confirmation popup is not prompted when the home button is clicked when config is false', function () {
```

Comment:
`'checks that the confirmation popup is prompted when the home button is clicked and the config is disabled'`

Reply:
Shouldn't it be "confirmation popup not prompted" when "config is disabled"?

Reply:
Yes sure.